tests/build/cythonize_pep420_namespace.srctree

```diff
@@ -10,7 +10,8 @@ from distutils.core import setup, Extension
 setup(
   ext_modules=cythonize([
     Extension("nsp.m1.a", ["nsp/m1/a.pyx"]),
-    Extension("nsp.m2.b", ["nsp/m2/b.pyx"])
+    Extension("nsp.m2.b", ["nsp/m2/b.pyx"]),
+    Extension("nsp.m3.c.d", ["nsp/m3/c/d.pyx"])
```

Comment:
It's generally a good idea to avoid having to change a previous line when a new line is added.
```suggestion
    Extension("nsp.m3.c.d", ["nsp/m3/c/d.pyx"]),
```